string hand
geometry_msgs/Vector3 location